kazoo/recipe/lock.py

```diff
@@ -106,6 +120,7 @@ def acquire(self, blocking=True, timeout=None):
         .. versionadded:: 1.1
             The timeout option.
         """
+        blocking = self.blocking if blocking else blocking
```

Comment:
I agree; I'd rather have this blocking take precedence over the instance one.

Otherwise it just seems confusing; maybe have ``blocking is None`` then use the instance one?